src/@next/Tag/TagStyle.ts

```diff
@@ -16,9 +17,25 @@ export const TagContentStyle = styled.span<TagContentProps>`
   }
 `;
 
+export const TagContentWrapper = styled.div`
+  padding: 0 ${space4};
+`;
+
 export const TagRemoveContainerStyle = styled.div<TagRemoveContainerProps>`
-  padding: ${space4};
+  padding: ${space4} ${space4} ${space4} 0;
   display: flex;
+
+  svg {
```

Comment:
this will affect all children that are svgs. Can you add a className on the Icon instead and apply the style based on it?

same with the Icon in the Tag content